source/initZones/locatorzonesV2.sqf

```diff
@@ -0,0 +1,91 @@
+_radius = _this select 0;
+_diff = _this select 1;
+
+[["The zones are being generated<br/>Please wait for mission initialization to complete..."]] spawn BIS_fnc_typeText;
+[[{player globalChat "DO NOT USE ACTION MENU UNTIL MISSION INITIALIZATION IS COMPLETE"}],"BIS_fnc_Spawn",true,false] call BIS_fnc_MP;
+[[{hint "DO NOT USE ACTION MENU UNTIL MISSION INITIALIZATION IS COMPLETE"}],"BIS_fnc_Spawn",true,false] call BIS_fnc_MP;
+
+
+// ON WATER ? >> FLAT ? >> DISTANCE BETWEEN ZONES ? >> OK !
+
+_worldName = getText(configFile >> "cfgWorlds" >> worldName >> "description");
+
+_fortified = false;
+openMap [true, true]; 
+hint "Click somewhere on the island to place enemy zones";
+_zones_array = [[-9999,-9999,-9999]];
+for [{_i=1}, {_i<=zones_number}, {_i=_i+1}] do  // BEGIN "FOR" LOOP --
+{
+    player globalChat format["Choose location for zone #%1...",_i];
+    _found = false;
+    _zone_radius = 0;
+    _points_zone = 0;
+    while {_zone_radius<=zones_min_radius} do {
+        _random_zones_max_radius = random zones_max_radius;
+        _zone_radius = round _random_zones_max_radius;   
+        _points_zone = round (_zone_radius/10);
+    };
+	
+    while {!_found} do {
+        clicked = false;
+        OnMapSingleClick "ClickedPos = _pos; clicked = true;";	
+        _found_distance = false;
+        _missionPos = [];
+
+        if (clicked) then {
+
+            _missionPos = [ClickedPos, 0,50,15,0,0.1,0,[],[[0,0],[0,0]]] call BIS_fnc_findSafePos;
+
+        if (0 == _missionPos select 0 && 0 == _missionPos select 1) then {
+            clicked = false;
+            hint "Invalid position, the position must be flat and no objects must be near the position";
+        } else {
+            onMapSingleClick "";
+            hint "Valid pos, creating enemy zone";
+            _found_distance = true;
+            };
```

Comment:
the code in the if here should be kicked over as well. I had to read this twice to catch what was off in the alignment. The start of the if here should be even with line 37 :-)